app/_src/kubernetes-ingress-controller/concepts/architecture.md

```diff
@@ -14,7 +14,17 @@ The {{site.kic_product_name}} configures {{site.base_gateway}} using Ingress or
 
 The figure illustrates how {{site.kic_product_name}} works:
 
-![high-level-design](/assets/images/products/kubernetes-ingress-controller/high-level-design.png "High Level Design")
+<!--vale off-->
+{% mermaid %}
```

Comment:
This seems to be getting a syntax error, but I don't know why since it displays just fine in the Mermaid live editor.